### PR TITLE
Source Stripe: external account streams

### PR DESCRIPTION
**Pull request by wjwatkinson**
## What
Adds two new streams to the stripe connector.

## How
Add two new streams

## 🚨 User Impact 🚨
Adds new streams

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details>
  <summary>Tests</summary>

```
Test session starts (platform: darwin, Python 3.9.13, pytest 6.2.5, pytest-sugar 0.9.4)
cachedir: .pytest_cache
rootdir: /Users/willwatkinson/src/github.com/airbytehq/airbyte, configfile: pytest.ini
plugins: sugar-0.9.4, requests-mock-1.9.3, timeout-1.4.2
collecting ...
 airbyte-integrations/connectors/source-stripe/unit_tests/test_source.py::test_lookback_window[None-1656694636.688665-1656694636.688665-if lookback_window_days is not set should not affect cursor value] ✓2% ▎
 airbyte-integrations/connectors/source-stripe/unit_tests/test_source.py::test_lookback_window[0-1656694636.688665-1656694636.688665-if lookback_window_days is not set should not affect cursor value] ✓5% ▌         {"type": "LOG", "log": {"level": "INFO", "message": "Applying lookback window of 10 days to stream invoices"}}

 airbyte-integrations/connectors/source-stripe/unit_tests/test_source.py::test_lookback_window[10-1656694636.688665-1655830636-Should calculate cursor value as expected] ✓7% ▊         {"type": "LOG", "log": {"level": "INFO", "message": "Applying lookback window of -10 days to stream invoices"}}

 airbyte-integrations/connectors/source-stripe/unit_tests/test_source.py::test_lookback_window[-10-1656694636.688665-1655830636-Should not care for the sign, use the module] ✓10% ▉
 airbyte-integrations/connectors/source-stripe/unit_tests/test_source.py::test_source_streams ✓12% █▎
 airbyte-integrations/connectors/source-stripe/unit_tests/test_source.py::test_source_check_connection_ok ✓14% █▌
 airbyte-integrations/connectors/source-stripe/unit_tests/test_source.py::test_source_check_connection_failure ✓17% █▋        {"type": "LOG", "log": {"level": "INFO", "message": "Applying lookback window of 1 days to stream checkout_sessions"}}
{"type": "LOG", "log": {"level": "INFO", "message": "Applying lookback window of 1 days to stream checkout_sessions"}}
{"type": "LOG", "log": {"level": "INFO", "message": "Applying lookback window of 1 days to stream checkout_sessions_line_items"}}
{"type": "LOG", "log": {"level": "WARN", "message": "{'error': {'code': 'resource_missing', 'doc_url': 'https://stripe.com/docs/error-codes/resource-missing', 'message': \"No such checkout session: 'cs_test_a165K4wNihuJlp2u3tknuohrvjAxyXFUB7nxZH3lwXRKJsadNEvIEWMUJ9'\", 'param': 'session', 'type': 'invalid_request_error'}}"}}
{"type": "LOG", "log": {"level": "INFO", "message": "Applying lookback window of 1 days to stream checkout_sessions_line_items"}}
stream_slice: {'checkout_session_id': 'cs_test_a1RjRHNyGUQOFVF3OkL8V8J0lZUASyVoCtsnZYG74VrBv3qz4245BLA1BP', 'expires_at': 100000}

 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_missed_id_child_stream ✓19% █▉
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_sub_stream ✓21% ██▎
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[Customers-kwargs0-customers] ✓24% ██▍
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[BalanceTransactions-kwargs1-balance_transactions] ✓26% ██▋
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[Charges-kwargs2-charges] ✓29% ██▉
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[CustomerBalanceTransactions-kwargs3-customers/C1/balance_transactions] ✓31% ███▏
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[Coupons-kwargs4-coupons] ✓33% ███▍
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[Disputes-kwargs5-disputes] ✓36% ███▋
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[Events-kwargs6-events] ✓38% ███▊
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[Invoices-kwargs7-invoices] ✓40% ████▏
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[InvoiceLineItems-kwargs8-invoices/I1/lines] ✓43% ████▍
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[InvoiceItems-kwargs9-invoiceitems] ✓45% ████▌
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[Payouts-kwargs10-payouts] ✓48% ████▊
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[Plans-kwargs11-plans] ✓50% █████
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[Products-kwargs12-products] ✓52% █████▎
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[Subscriptions-kwargs13-subscriptions] ✓55% █████▌
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[SubscriptionItems-kwargs14-subscription_items] ✓57% █████▊
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[Transfers-kwargs15-transfers] ✓60% █████▉
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[Refunds-kwargs16-refunds] ✓62% ██████▎
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[PaymentIntents-kwargs17-payment_intents] ✓64% ██████▌
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[BankAccounts-kwargs18-customers/C1/sources] ✓67% ██████▋
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[CheckoutSessions-kwargs19-checkout/sessions] ✓69% ██████▉
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[CheckoutSessionsLineItems-kwargs20-checkout/sessions/CS1/line_items] ✓71% ███████▎
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[PromotionCodes-kwargs21-promotion_codes] ✓74% ███████▍
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_path[ExternalAccount-kwargs22-accounts/<account_id>/external_accounts] ✓76% ███████▋
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_request_params[CustomerBalanceTransactions-kwargs0-expected0] ✓79% ███████▉
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_request_params[Customers-kwargs1-expected1] ✓81% ████████▏
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_request_params[InvoiceLineItems-kwargs2-expected2] ✓83% ████████▍
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_request_params[Subscriptions-kwargs3-expected3] ✓86% ████████▋
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_request_params[SubscriptionItems-kwargs4-expected4] ✓88% ████████▊
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_request_params[BankAccounts-kwargs5-expected5] ✓90% █████████▏{"type": "LOG", "log": {"level": "INFO", "message": "Applying lookback window of 1 days to stream checkout_sessions"}}

 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_request_params[CheckoutSessions-kwargs6-expected6] ✓93% █████████▍{"type": "LOG", "log": {"level": "INFO", "message": "Applying lookback window of 1 days to stream checkout_sessions_line_items"}}

 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_request_params[CheckoutSessionsLineItems-kwargs7-expected7] ✓95% █████████▌
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_request_params[ExternalAccountBankAccounts-kwargs8-expected8] ✓98% █████████▊
 airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py::test_request_params[ExternalAccountCards-kwargs9-expected9] ✓100% ██████████
=============================== warnings summary ===============================
airbyte-integrations/connectors/source-stripe/unit_tests/test_source.py: 28 warnings
airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py: 37 warnings
  /Users/willwatkinson/src/github.com/airbytehq/airbyte/airbyte-integrations/connectors/source-stripe/.venv/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py:41: DeprecationWarning: Call to deprecated class NoAuth. (Set `authenticator=None` instead) -- Deprecated since version 0.1.20.
    self._authenticator: HttpAuthenticator = NoAuth()

airbyte-integrations/connectors/source-stripe/unit_tests/test_source.py: 29 warnings
airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py: 37 warnings
  /Users/willwatkinson/src/github.com/airbytehq/airbyte/airbyte-integrations/connectors/source-stripe/.venv/lib/python3.9/site-packages/deprecated/classic.py:173: DeprecationWarning: Call to deprecated class HttpAuthenticator. (Use requests.auth.AuthBase instead) -- Deprecated since version 0.1.20.
    return old_new1(cls, *args, **kwargs)

airbyte-integrations/connectors/source-stripe/unit_tests/test_source.py::test_source_streams
  /Users/willwatkinson/src/github.com/airbytehq/airbyte/airbyte-integrations/connectors/source-stripe/source_stripe/source.py:52: DeprecationWarning: Call to deprecated class TokenAuthenticator. (Use airbyte_cdk.sources.streams.http.requests_native_auth.TokenAuthenticator instead) -- Deprecated since version 0.1.20.
    authenticator = TokenAuthenticator(config["client_secret"])

-- Docs: https://docs.pytest.org/en/stable/warnings.html

Results (1.41s):
      42 passed
```

</details>

### CODE REVIEW

**Comment by pratik2315** [comment]
Hey there! Is this issue open to work on?

File: airbyte-integrations/connectors/source-stripe/README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Customize `acceptance-test-config.yml` file to configure tests. See [Source Acce
 If your connector requires to create or destroy resources for use during acceptance tests create fixtures for it and place them inside integration_tests/acceptance.py.
 To run your integration tests with acceptance tests, from the connector root, run
 ```
-python -m pytest integration_tests -p integration_tests.acceptance
+sh ./acceptance-test-docker.sh
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-stripe/README.md:69]: @wjwatkinson can revert this change here?

**Comment by marcosmarxm** [comment]
@wjwatkinson can you provide the integration tests too?

**Comment by marcosmarxm** [comment]
The integration account doesn't have access to both endpoints. @wjwatkinson need your output to confirm it's working. 

```
***"type": "TRACE", "trace": ***"type": "ERROR", "emitted_at": 1656956516330.858, "error": ***"message": "Only live keys can access this method.", "internal_message": "400 Client Error: Bad Request for url: https://api.stripe.com/v1/accounts/***/external_accounts?limit=100&object=bank_account", "stack_trace": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py\", line 115, in read\n    yield from self._read_stream(\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py\", line 165, in _read_stream\n    for record in record_iterator:\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py\", line 254, in _read_full_refresh\n    for record in records:\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 417, in read_records\n    response = self._send_request(request, request_kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 336, in _send_request\n    return backoff_handler(user_backoff_handler)(request, request_kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/backoff/_sync.py\", line 105, in retry\n    ret = target(*args, **kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/backoff/_sync.py\", line 105, in retry\n    ret = target(*args, **kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 303, in _send\n    raise exc\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 300, in _send\n    response.raise_for_status()\n  File \"/usr/local/lib/python3.9/site-packages/requests/models.py\", line 1021, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\nrequests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://api.stripe.com/v1/accounts/***/external_accounts?limit=100&object=bank_account\n", "failure_type": "system_error"***
```

**Comment by wjwatkinson** [comment]
@marcosmarxm those endpoints don't work when stripe is in test mode (400 error), which I assume the credentials are. Here are passing integration tests for the `ExternalAccountBankAccounts` stream:
<details>

```
sh ./acceptance-test-docker.sh
[+] Building 0.3s (12/12) FINISHED
 => [internal] load build definition from Dockerfile                                                                                         0.0s
 => => transferring dockerfile: 37B                                                                                                          0.0s
 => [internal] load .dockerignore                                                                                                            0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [internal] load metadata for docker.io/library/python:3.9-slim                                                                           0.2s
 => [1/7] FROM docker.io/library/python:3.9-slim@sha256:c01a2db78654c1923da84aa41b829f6162011e3a75db255c24ea16fa2ad563a0                     0.0s
 => [internal] load build context                                                                                                            0.0s
 => => transferring context: 1.83kB                                                                                                          0.0s
 => CACHED [2/7] RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*                                                0.0s
 => CACHED [3/7] WORKDIR /airbyte/integration_code                                                                                           0.0s
 => CACHED [4/7] COPY source_stripe ./source_stripe                                                                                          0.0s
 => CACHED [5/7] COPY main.py ./                                                                                                             0.0s
 => CACHED [6/7] COPY setup.py ./                                                                                                            0.0s
 => CACHED [7/7] RUN pip install .                                                                                                           0.0s
 => exporting to image                                                                                                                       0.0s
 => => exporting layers                                                                                                                      0.0s
 => => writing image sha256:ff64d7cbae1ca499778ce0b8e08e7b214cf3fb6c3796d5d192ae99417cf70162                                                 0.0s
 => => naming to docker.io/airbyte/source-stripe                                                                                             0.0s
latest: Pulling from airbyte/source-acceptance-test
Digest: sha256:ba4d7efa790d2356ad84a6e053d4878a12d7a054f1c5992bd38e6cf1f930036a
Status: Image is up to date for airbyte/source-acceptance-test:latest
docker.io/airbyte/source-acceptance-test:latest
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
Test session starts (platform: linux, Python 3.9.11, pytest 6.2.5, pytest-sugar 0.9.4)
rootdir: /test_input
plugins: timeout-1.4.2, sugar-0.9.4
collecting ...
 test_core.py ✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓                                                                                                  95% █████████▌
 test_full_refresh.py ✓                                                                                                            100% ██████████

============================================================ short test summary info =============================================================
SKIPPED [1] source_acceptance_test/plugin.py:56: Skipping TestIncremental.test_two_sequential_reads because not found in the config

Results (11.18s):
      21 passed
```

</deails>

I don't have an account where an external account card is set up, so I can't get that test to pass. Given these endpoints only work in production I don't have access to any accounts where I can add a card.

**Comment by wjwatkinson** [comment]
@marcosmarxm I can remove ExternalAccountCards, but given it's just a parameter on the same endpoint it seemed worth including. That said it does have a different schema, so I understand not wanting to add it if I can't test it.

**Review by marcosmarxm — APPROVED** [review]
Looks good to me @wjwatkinson thanks for the contribution. I asked also to connector team review this because Stripe is a GA connector.

**Comment by grubberr** [comment]
As for me PR looks good.
One way we can improve reliability of this process is to ask @wjwatkinson to run SAT tests himself.
As I see that @wjwatkinson already did it but I am not sure if he added new streams `external_account_bank_accounts` and `external_account_cards` to `integration_tests/full_refresh_configured_catalog.json` before running tests.

if not

We can add this new stream(s) to catalog:

```diff
diff --git a/airbyte-integrations/connectors/source-stripe/integration_tests/full_refresh_configured_catalog.json b/airbyte-integrations/connectors/source-stripe/integration_tests/full_refresh_configured_catalog.json
index a6fbc37c8e..169e156f7e 100644
--- a/airbyte-integrations/connectors/source-stripe/integration_tests/full_refresh_configured_catalog.json
+++ b/airbyte-integrations/connectors/source-stripe/integration_tests/full_refresh_configured_catalog.json
@@ -44,6 +44,17 @@
       "destination_sync_mode": "overwrite",
       "cursor_field": ["checkout_session_expires_at"],
       "primary_key": [["id"]]
+    },
+    {
+      "stream": {
+        "name": "external_account_bank_accounts",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh"],
+        "source_defined_primary_key": [["id"]]
+      },
+      "sync_mode": "full_refresh",
+      "destination_sync_mode": "overwrite",
+      "primary_key": [["id"]]
     }
   ]
 }
```

run SAT tests on @wjwatkinson laptop, revert commit if it was commited

**Comment by marcosmarxm** [comment]
> As for me PR looks good. One way we can improve reliability of this process is to ask @wjwatkinson to run SAT tests himself. As I see that @wjwatkinson already did it but I am not sure if he added new streams `external_account_bank_accounts` and `external_account_cards` to `integration_tests/full_refresh_configured_catalog.json` before running tests.
> 
> if not
> 
> We can add this new stream(s) to catalog:
> 
> ```diff
> diff --git a/airbyte-integrations/connectors/source-stripe/integration_tests/full_refresh_configured_catalog.json b/airbyte-integrations/connectors/source-stripe/integration_tests/full_refresh_configured_catalog.json
> index a6fbc37c8e..169e156f7e 100644
> --- a/airbyte-integrations/connectors/source-stripe/integration_tests/full_refresh_configured_catalog.json
> +++ b/airbyte-integrations/connectors/source-stripe/integration_tests/full_refresh_configured_catalog.json
> @@ -44,6 +44,17 @@
>        "destination_sync_mode": "overwrite",
>        "cursor_field": ["checkout_session_expires_at"],
>        "primary_key": [["id"]]
> +    },
> +    {
> +      "stream": {
> +        "name": "external_account_bank_accounts",
> +        "json_schema": {},
> +        "supported_sync_modes": ["full_refresh"],
> +        "source_defined_primary_key": [["id"]]
> +      },
> +      "sync_mode": "full_refresh",
> +      "destination_sync_mode": "overwrite",
> +      "primary_key": [["id"]]
>      }
>    ]
>  }
> ```
> 
> run SAT tests on @wjwatkinson laptop, revert commit if it was commited

@grubberr did you check the comment here: https://github.com/airbytehq/airbyte/pull/14357#issuecomment-1175165669

**Comment by grubberr** [comment]
@wjwatkinson can you modify `integration_tests/full_refresh_configured_catalog.json` as I provided on example and run Source Acceptance Test on your laptop ?

**Comment by marcosmarxm** [comment]
@grubberr I'll be on vacation do you mind finish and publish this after is ready?

**Review by grubberr — APPROVED** [review]

File: airbyte-integrations/connectors/source-stripe/source_stripe/schemas/external_account_bank_accounts.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      "type": ["string", "null"]
+    },
+    "fingerprint": {
+      "type": ["string", "null"]
+    },
+    "last4": {
+      "type": ["string", "null"]
+    },
+    "metadata": {
+      "type": "object"
```
Comment (grubberr) [inline airbyte-integrations/connectors/source-stripe/source_stripe/schemas/external_account_bank_accounts.json:36]: Do we need to add "null" here ? Because I see on stream `external_account_cards`  we have 
```
 "metadata": {
      "type": [
        "object",
        "null"
      ]
},
```

File: airbyte-integrations/connectors/source-stripe/source_stripe/schemas/external_account_cards.json [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "object": {
+      "type": [
+        "string",
+        "null"
```
Comment (grubberr) [inline airbyte-integrations/connectors/source-stripe/source_stripe/schemas/external_account_cards.json:11]: Can you please format this one line :

```
type": [ "string", "null"]
```

to be consisten with other schema files

**Comment by grubberr** [comment]
@marcosmarxm I have added a couple of notes 
please @wjwatkinson fix them 
and I think we can publish and merge

File: airbyte-integrations/connectors/source-stripe/source_stripe/schemas/external_account_cards.json (hunk 3 above)
Comment (wjwatkinson) [reply airbyte-integrations/connectors/source-stripe/source_stripe/schemas/external_account_cards.json:11]: Not sure what is happening here. My commit changed this [as is visible in the diff](https://github.com/airbytehq/airbyte/pull/14357/commits/3b0812738dfa2d21b500e091d84d64fd3e4fac5b), but it doesn't seem to be taking effect here.

**Comment by grubberr** [comment]
as for me all good I am ready to publish
@evantahler please also look
to have 2 reviews from airbyte team because it's GA connector

**Review by evantahler — APPROVED** [review]
Given that 
> Given these endpoints only work in production...

I guess there isn't much testing we can add.  Normally, as this is a GA connector, we would want to at least add some "expected_records" for SAT testing.

**Comment by marcosmarxm** [comment]
> 

But expected records match records returned from the API in integration tests or are you asking to match records in unit test?
